pkg/datagen/datagen.go

```diff
@@ -37,29 +37,32 @@ type Poly struct {
 }
 
 type Doc struct {
-	Key      string `json:"_key"`
-	Sha      string `json:"sha"`
-	Label    string `json:"label,omitempty"`
-	From     string `json:"_from,omitempty"`
-	To       string `json:"_to,omitempty"`
-	Payload0 string `json:"payload0"`
-	Payload1 string `json:"payload1,omitempty"`
-	Payload2 string `json:"payload2,omitempty"`
-	Payload3 string `json:"payload3,omitempty"`
-	Payload4 string `json:"payload4,omitempty"`
-	Payload5 string `json:"payload5,omitempty"`
-	Payload6 string `json:"payload6,omitempty"`
-	Payload7 string `json:"payload7,omitempty"`
-	Payload8 string `json:"payload8,omitempty"`
-	Payload9 string `json:"payload9,omitempty"`
-	Payloada string `json:"payloada,omitempty"`
-	Payloadb string `json:"payloadb,omitempty"`
-	Payloadc string `json:"payloadc,omitempty"`
-	Payloadd string `json:"payloadd,omitempty"`
-	Payloade string `json:"payloade,omitempty"`
-	Payloadf string `json:"payloadf,omitempty"`
-	Geo      *Poly  `Json:"geo,omitempty"`
-	Words    string `json:"words,omitempty"`
+	Key       string `json:"_key"`
+	Index     string `json: "index"`
```

Comment:
Should we also put `omitempty` here?

Reply:
I'll strike Index, actually no need for it since we have `Key`.

Reply:
Index is still needed, but it's a uint64, used for internal needs. Is it correct to omit the tag if I don't want to marshal it? (Do I, actually?)

Reply:
If you do not want to marshal a field, you should use `json: "-"` to explicitly say so.